script_library/deployment-actions-kvm.sh

```diff
@@ -56,7 +56,12 @@ function deploy_airship(){
 }
 function add_compute(){
     echo "Now Add compute"
-    run_ansible ${socok8s_absolute_dir}/playbooks/add_compute.yml
+    tagged_info=''
+    tags='all'
+    if [[ "${1:-default}" != 'default' ]]; then
+        tagged_info=" --tags $1"
+    fi
+    run_ansible ${socok8s_absolute_dir}/playbooks/add_compute.yml ${tagged_info}
```

Comment:
Do you need that additional logic ? Just directly pass whatever tag ( add_computenode) is used to identify add_compute node steps ? Not sure what other tags can be passed to add_compute function ?